test/isInt.js

```diff
@@ -0,0 +1,40 @@
+var PushStateTree = require('../src/pushStateTree');
+
+describe('PushStateTree isInt', function() {
+
+  it('should return true for 0', function() {
```

Comment:
`should return true for number values`, then you can expect true for 0 and 1 in the same test